src/Type/BasePickerType.php

```diff
@@ -46,38 +45,19 @@ abstract class BasePickerType extends AbstractType
      */
     private $formatConverter;
 
-    public function __construct(MomentFormatConverter $formatConverter, $translator, ?RequestStack $requestStack = null)
+    public function __construct(MomentFormatConverter $formatConverter, TranslatorInterface $translator, ?RequestStack $requestStack = null)
     {
-        if (!$translator instanceof LegacyTranslatorInterface && !$translator instanceof TranslatorInterface) {
+        if (null === $requestStack) {
             throw new \InvalidArgumentException(sprintf(
-                'Argument 2 should be an instance of %s or %s',
-                LegacyTranslatorInterface::class,
-                TranslatorInterface::class
+                'Argument 3 should be an instance of %s',
+                RequestStack::class
             ));
```

Comment:
:thinking: the same reasoning applies here… Let's make `RequestStack` mandatory in the signature, shall we?